client/my-sites/plugins/plugin-settings-link/README.md

```diff
@@ -0,0 +1,18 @@
+Plugin Settings Link
+=====================
+
+This component is used to display a link that take the user to the plugins settings section. Where they can set up the plugin.
```

Comment:
I think this should sound better without a period there. Like "This component is used to display a link that take the user to the plugins settings section, where they can set up the plugin."